pipes/WDL/workflows/tasks/tasks_metagenomics.wdl

```diff
@@ -46,6 +46,13 @@ task krakenuniq {
 
     wait # for krona_taxonomy_db_tgz to download and extract
 
+    metagenomics.py taxlevel_summary \
+      `cat $OUT_REPORTS` \
+      --csvOut aggregate_viral_summary.csv \
+      --noHist --taxHeading Viruses \
```

Comment:
can we make taxHeading an optional WDL String input that defaults to "Viruses"?

Reply:
yes, of course. I'll parameterize this a bit more.